modules/hitlnext/src/backend/repository.ts

```diff
@@ -464,4 +472,34 @@ export default class Repository {
       { count: conditions.limit, sortOrder: [{ column: 'id', desc: true }] }
     )
   }
+
+  deleteHandoff = async (id: string) => {
+    return this.bp.database.transaction(async trx => {
+      const handoff = await this.getHandoff(id)
+
+      // We want delete the events since it contains
+      // the user's conversation.
+      await trx(EVENT_TABLE_NAME)
```

Comment:
that will delete the whole user and agent conversation (even all the events pre-handoff) is that what we want ?

I think we want to delete all the events related to a handoff . Or is it ?

Reply:
You're right! I added a condition so only the events after the handoff is created will be removed.

Reply:
My 2 cents from working privacy requirements is that you would want to erase everything. I don't really see a use case for erasing only pre or post handoff.